### PR TITLE
Saas 12.5 social feedback awa

### PR DESCRIPTION
**Pull request by awa-odoo**
To merge without merge commit

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/link_tracker/models/link_tracker.py [hunk 1]
```diff
@@ -200,6 +200,14 @@ def action_visit_page(self):
             'target': 'new',
         }
 
+    def action_visit_page_statistics(self):
```
Comment (tde-banana-odoo) [inline addons/link_tracker/models/link_tracker.py]: Dependent of website_links where the route '+' (so much naming) is defined ... so to move in that module.

File: addons/website_sms/views/website_visitor_views.xml [hunk 2]
```diff
@@ -32,4 +32,17 @@
             </xpath>
         </field>
     </record>
+
+     <record id="website_visitor_view_tree" model="ir.ui.view">
+        <field name="name">website.visitor.view.tree.inherit.website.mass.mailing.sms</field>
```
Comment (tde-banana-odoo) [inline addons/website_sms/views/website_visitor_views.xml]: website mass mailing sms ? seems dependent on sms only.

File: addons/link_tracker/models/link_tracker.py (hunk 1 above)
Comment (awa-odoo) [reply addons/link_tracker/models/link_tracker.py]: @qmo-odoo Can you fix it ?

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff

**Comment by KangOl** [comment]
@robodoo p=1

**Pull request closed without merging**